### ISSUE TITLE
Emotet Malware Analysis

### ISSUE THREAD

**Issue by omergunal** [LetsDefend/Blue-Team-Community#2]
This challenge will be release in the "Malware Analysis" section at https://app.letsdefend.io/challenge/

**Steps:**
1. A small attack scenario should be prepared
2.  A malware related to the Emotet must be created/found
3. At least 7-10 questions should be prepared about the activities of the malware.
4. Each questions should has a useful hint
5. Question answers should be clear. An example question: What is the MD5 value of Malware?
6. (Optional) _Create a thumbnail for listing, 400x300, PNG format_
7. Create a walkthrough about your challenge
8. Upload everything on Google Drive and send the link to us as a comment or to info@letsdefend.io
9. LetsDefend team will review your materials. If necessary, will do some edits for a better format then release it.

**Basically, you need:**

- Malware sample
- Questions, answers, and hints
- Walkthrough 

**Useful links:**

- https://any.run/malware-trends/emotet
- Example challenge: https://app.letsdefend.io/challenge/pdf-analysis/
